cmd/di.go

```diff
@@ -948,7 +948,7 @@ func (di *Dependencies) bootstrapLocationComponents(options node.Options) (err e
 
 	di.LocationResolver = location.NewCache(resolver, di.EventBus, time.Minute*5)
 
-	if !config.GetBool(config.FlagProxyMode) {
+	if !config.GetBool(config.FlagProxyMode) && !config.GetBool(config.FlagDVPNMode) {
```

Comment:
Should it be `||`?

Reply:
we subscribe to that topic only if node running not in `proxymode` AND not in `dvpnmode`. that's why it should be `&&`